src/ripple/beast/container/detail/aged_ordered_container.h

```diff
@@ -328,7 +328,7 @@ class aged_ordered_container
         {
         }
 
-        config_t (config_t&& other)
+        config_t (config_t&& other) noexcept
```

Comment:
I believe this may call `terminate`, and I was unable to fix it by making `KeyValueCompare` `noexcept`, because that looked like it will call `terminate`.  Thus everything in this header fails without this foundation.

`aged_ordered_container` could probably be made `noexcept`, but not this easily.

Reply:
Hold off on this.  I think I see a way to make this work...

Reply:
<sigh>  I can make `config_t` work.  But `cont_type` and `chronological.list` are boost::intrusive types and they are not nothrow_move_constructible and nothrow_move_assignable respectively.  That means the ordered_container move constructor can throw.

Reply:
@HowardHinnant I definitely overlooked the fact that if we can not prove that the whole call-def-chain is `noexcept` for all inputs, we can't mark the top function in the chain as `noexcept`. You're exactly right about the worrisome calls to `std::terminate`.

In this particular case, `boost::intrusive` types are the showstopper as you pointed out. I'm not even sure there is value in making `config_t` `noexcept-correct`, so I may just revert.